tools/generate-operator-patches.ts

```diff
@@ -57,19 +58,26 @@ const AliasMethodOverrides = {
 };
 
 function generateNewOperatorFileContents (op:OperatorWrapper): OperatorWrapper {
+  var header = `/**
```

Comment:
Sure. I'll change this one. I did not touch the others because they're not related to the patch.